code/code/misc/combat.cc

```diff
@@ -1992,9 +1994,9 @@ int TBeing::getWeaponDam(const TBeing *v, const TThing *wielded, primaryTypeT is
     // 2h Specialization: should be no additional dam for anyone without specialization
     // extra 10% for 2h is part of the object
     TObj *obj = dynamic_cast<TObj *>(heldInPrimHand());
-    if (obj && obj->isPaired() && doesKnowSkill(SKILL_2H_SPEC_DEIKHAN)) {
+    if (obj && obj->isPaired() && doesKnowSkill(SKILL_2H_SPEC)) {
```

Comment:
Need getSkillNum here too

Reply:
Updated - hope this is the correct usage.  Please let me know if I'm missing something.